src/elements/content-sidebar/ActivitySidebar.js

```diff
@@ -342,12 +311,11 @@ class ActivitySidebar extends React.PureComponent<Props, State> {
     };
 
     updateTaskAssignment = (taskId: string, taskAssignmentId: string, status: TaskCollabStatus): void => {
-        const { file, api, onTaskAssignmentUpdate } = this.props;
-        const { currentUser = {} } = this.state;
+        const { api, currentUser, file, onTaskAssignmentUpdate } = this.props;
 
         const successCallback = () => {
             this.feedSuccessCallback();
-            onTaskAssignmentUpdate(taskId, taskAssignmentId, status, currentUser.id);
+            onTaskAssignmentUpdate(taskId, taskAssignmentId, status, currentUser?.id);
```

Comment:
Is it possible for `currentUser` to be undefined. Is that desirable or should this throw? Also, we don't see much optional chaining in BUIE, at least in top-level Elements. Does this present any issues for the build?